libs/durability/disk_failure_recovery_libs.py

```diff
@@ -287,33 +289,35 @@ def perform_near_full_sys_writes(s3userinfo, user_data_writes, bucket_prefix: st
             workload.extend([1024,2048,3072,4096])
 
         mb = 1024 * 1024
-        client = 10
         workload = [each * mb for each in workload]  # convert to bytes
         each_workload_byte = user_data_writes / len(workload)
         return_list = []
         for obj_size in workload:
             samples = int(each_workload_byte / obj_size)
             if samples > 0:
+                temp_client = client
+                if temp_client > samples:
+                    temp_client = samples
                 bucket_name = f'{bucket_prefix}-{obj_size}b'
                 obj_name = f'obj_{obj_size}'
                 resp = s3bench.s3bench(s3userinfo['accesskey'],
                                        s3userinfo['secretkey'],
                                        bucket=bucket_name,
-                                       num_clients=client,
+                                       num_clients=temp_client,
                                        num_sample=samples,
-                                       obj_name_pref=obj_name, obj_size=obj_size,
+                                       obj_name_pref=obj_name, obj_size=f"{obj_size}b",
                                        skip_cleanup=True, duration=None,
                                        log_file_prefix=f"workload_{obj_size}mb",
```

Comment:
here workload is in byte, we can change mb to b or convert the bytes to mb again.

Reply:
fixed